constants.js

```diff
@@ -0,0 +1,5 @@
+const APPLICATION_NAME = process.env['APPLICATION_NAME'] || '';
+const INSTANCE_ID = process.env['INSTANCE_ID'] || '';
+const INSTANCE_NAME = process.env['INSTANCE_NAME'] || '';
```

Comment:
This means we rely (or offer the feature) on env variable for more complete logs.
So we should add a small section in the readme explaining what the logs looks like and that we can inject APPLICATION_NAME, INSTANCE_ID and INSTANCE_NAME in them.

Reply:
Done